homeassistant/components/yale_smart_alarm/lock.py

```diff
@@ -0,0 +1,125 @@
+"""Support for Yale Lock."""
```

Comment:
This is a new platform, which should be in a separate PR.
Please remove it from this PR.

Reply:
Ok. So just to confirm I remove the added platforms from this PR, add it back into new PR's and then reference this PR from those.
Sorry for asking but wasn't clear I needed to do separate PR's for the added platforms and of course then this PR needs to be approved first before the ones for the added platforms.

Reply:
PR should be kept as small as possible, and preferably change only a single feature or thing. This is documented on our developer site and is also mentioned in the PR template for this PR.

For that matter, even if the dependency bump can be done separately, it should even become a separate PR.